test/pages/business/CountryOfResidenceInTheUkYesNoPageSpec.scala

```diff
@@ -32,7 +32,7 @@ class CountryOfResidenceInTheUkYesNoPageSpec extends PageBehaviours {
     beRemovable[Boolean](CountryOfResidenceYesNoPage)
 
     "implement cleanup logic when YES selected" in {
-      val userAnswers = UserAnswers("id", "identifier", "sessionId", LocalDate.now)
+      val userAnswers = UserAnswers("id", "identifier", "sessionId", "newId", LocalDate.now)
```

Comment:
this is wrong, maybe it is working in tests but it is not doing the same logic as in repository, you are suppose to use combination of id and utr and sessionId

Reply:
done